Packages/Env/Sources/Env/UserPreferences.swift

```diff
@@ -25,6 +25,7 @@ public class UserPreferences: ObservableObject {
   @AppStorage("app_auto_expand_spoilers") public var appAutoExpandSpoilers = false
   @AppStorage("app_auto_expand_media") public var appAutoExpandMedia: ServerPreferences.AutoExpandMedia = .hideSensitive
   @AppStorage("app_default_post_visibility") public var appDefaultPostVisibility: Models.Visibility = .pub
+  @AppStorage("app_default_reply_visibility") public var appDefaultReplyVisibility: Models.Visibility = .unlisted
```

Comment:
What do you think of this default? I feel like if there was no setting before it should be public. Or at least be initialised the first time with appDefaultPostVisibility